libraries/DataFlash/DataFlash_File.h

```diff
@@ -124,19 +122,19 @@ class DataFlash_File : public DataFlash_Backend
 
     void _io_timer(void);
 
-    uint16_t critical_message_reserved_space() const {
+    uint32_t critical_message_reserved_space() const {
         // possibly make this a proportional to buffer size?
-        uint16_t ret = 1024;
-        if (ret > _writebuf_size) {
+        uint32_t ret = 1024;
+        if (ret > _writebuf.get_size()) {
             // in this case you will only get critical messages
-            ret = _writebuf_size;
+            ret = _writebuf.get_size();
         }
         return ret;
     };
     uint16_t non_messagewriter_message_reserved_space() const {
         // possibly make this a proportional to buffer size?
         uint16_t ret = 1024;
-        if (ret >= _writebuf_size) {
+        if (ret >= _writebuf.get_size()) {
```

Comment:
*get_size* returns *uint32_t* so I would prefer that, like the previous method, this is changed to return an *uint32_t* even if it isn't used for now.